archlinuxcn/python-torchgan-git/lilac.yaml

```diff
@@ -0,0 +1,14 @@
+maintainers:
+  - github: hubutui
+
+build_prefix: extra-x86_64
+
+pre_build: vcs_update
+
+post_build: git_pkgbuild_commit
+
+depends:
+  - python-torchvision
+
+update_on:
+  - github: torchgan/torchgan
```

Comment:
It would be better to add an items `- alias: python`, so it can be automaticlly rebuilt when bumping python version.